README.md

```diff
@@ -157,8 +157,8 @@ make terraform
 will display a plan and ask you to confirm it. Please consult the Terraform
 documentation for details. You will need to run `make terraform` once to set up
 your deployment and every time code changes define new cloud resources. The
-resources are defined in `….tf.json` files which in turn are generated from
-`….tf.json.template.py` files which are simple Python scripts containing the
+resources are defined in `...tf.json` files which in turn are generated from
+`....tf.json.template.py` files which are simple Python scripts containing the
 desired JSON as Python dictionary and list literals and comprehensions.
```

Comment:
Remove this change.

Reply:
So you changed your mind about the ellipsis?